eap-datagrid-subsystem/README.md

```diff
@@ -0,0 +1,100 @@
+
+        $EAP_HOME/bin/standalone.sh
+
+4. use the CLI scipts to configure the datagrid subsystem and necessary dependencies
+
+        $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install1-standalone-local.cli
+        > restart the server !
+       $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install2-standalone-local.cli
+
+
+Build and Run the example
+-------------------------
+1. Type this command to build and deploy the archive:
+
+        mvn clean package
+        cp web/target/jboss-eap-datagrid-subsystem-AppWeb.war $EAP_HOME/standalone/deployments
+
+
+Run the web application in your browser
+=======================================
+
+      http://localhost:8080/jboss-eap-datagrid-subsystem-AppWeb/
+
+Add, delete and list entries ...
+
+Run as a clustered cache
+=====================================
+
+The configuration for EAP is a not clustered one, but it is possible to use the install scripts for a *ha* configuration as well.
+If the JDG caches should be clustered the installation scripts in the step above should be install?-standalone-clustered.cli
+But the JDG caches are clustered in any case, there is no need for EAP to be clustered.
+
+To see the behaviour for JDG caches
+
+1.  Create a fresh copy of EAP 7.1 (or newer) server
+
+2.  Unzip jboss-datagrid-7.2.0-eap-modules-library.zip and copy the modules/* directories into the EAP_HOME/modules directory
+
+3.  start EAP with the default configuration standalone.xml
+
+        $EAP_HOME/bin/standalone.sh [-c standalone-ha.xml]
+
+4. use the CLI scipts to configure the datagrid subsystem and necessary dependencies
+
+        $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install1-standalone-clustered.cli
+        > restart the server !
+       $EAP_HOME/bin/jboss-cli.sh -c --file=$SRC_HOME/install2-standalone-clustered.cli
+
+5. Stop the server
+
+6. Copy the web application to the deployment folder
+
+        cp web/target/jboss-eap-datagrid-subsystem-AppWeb.war $EAP_HOME/standalone/deployments
+
+7. Copy the EAP instance to another directory
+
+8. Start both servers and use the property to change the JDG multicast address
+
+        $EAP_HOME1/bin/standalone.sh [-c standalone-ha.xml] -Djdg.bind.address=<IP address>
+        $EAP_HOME21/bin/standalone.sh [-c standalone-ha.xml] -Djdg.bind.address=<IP address> [-Djboss.socket.binding.port-offset=100]
+
+9.  Access both web applications and check that the cache entries are distributed
+
+
+Notes
+=======
+  - You can simple rename the jboss-eap-datagrid-subsystem-AppWeb.war to any other name.war to simulate different applications
+    The applicatations will share the same cache
+  - The cache will be independent from the application lifecycles, as well as you undeploy all applications
+    Even if the cache has no persistence the data will be kept as long as the server is not stopped
+  - In this example the EAP server is not clustered, but the cache include the JGroups subsystem for clustering
```

Comment:
"but the JGroups subsystem is utilised in order for cache instances to be clustered."

Tbh I'm not 100% on what you are trying to say in this bullet point.